benches/example.rs

```diff
@@ -54,6 +54,7 @@ fn file_path(file_name: &str) -> path::PathBuf {
 
 fn parsing_tagger<'a>(kinds: &'a [OutputKind], context: &'a IdentityContext<Dimension>) -> CandidateTagger<'a, IdentityContext<Dimension>> {
     CandidateTagger {
+        input: "input",
```

Comment:
What is this `input` for ?

Reply:
That was to track the text match of the candidate, but as it's mostly for debug, it's now removed.